configs/nemo_configs/sft_megatron_20b.yaml

```diff
@@ -0,0 +1,148 @@
+name: megatron_gpt
+restore_from_path: null  # used when starting from a .nemo file
+
+trainer:
+  devices: 8
+  num_nodes: 4
+  accelerator: gpu
+  precision: bf16
```

Comment:
did u notice any issue w bf16 vs fp16?

Reply:
From a few trial runs I noticed that fp16 was a bit more unstable (very large grad norms at the start - see [trlx: BF16 v FP16 - Add Supervised Fine-Tuning \(SFT\) support for NeMo backend \#353](https://api.wandb.ai/links/carperai/xyfpdy3r))  so I opted to default to bf16.

Reply:
oh okay thanks for checking :)